### PR TITLE
Dtspb 2433 ee yes journey

### PR DESCRIPTION
**Pull request by nigeldunne**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSPB-2433

### Change description ###

EE yes journey pages

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: app/resources/cy/translation/iht/estateform.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "title": "Which IHT forms did you complete to report the estate&rsquo;s value?",
+  "question": "Which IHT forms did you complete to report the estate&rsquo;s value?",
+  "optionIHT400421": "IHT 400 and 421",
+  "optionIHT400421Hint": "Forms you completed because you have to pay inheritance tax",
+  "optionIHT207": "IHT 207",
+  "optionIHT207Hint": "Forms you completed because you do not have to pay inheritance tax because it&rsquo;s an excepted estate and the person who&rsquo;s died was domiciled abroad",
+  "errors": {
+    "ihtFormEstateId": {
+      "required": "Select which IHT forms you completed to report the estate’s value"
```
Comment (AbdullahShahHMCTS) [inline app/resources/cy/translation/iht/estateform.json:10]: escape the apostrophe

File: app/resources/cy/translation/iht/probateestatevalues.json [hunk 2]
```diff
@@ -1,32 +1,29 @@
...
-    "poundSymbol": "&pound;",
+  "title": "IHT-207",
+  "question207": "What are the values of the estate you entered on form IHT 207?",
+  "question421": "What are the values of the estate you entered on form IHT 421?",
+  "questionNoIHT": "What are the values of the estate for probate?",
+  "hint207": "<br>You can find these values on the completed IHT 207 form.<br><br><br>",
+  "hint421": "<br>You can find these values on the completed IHT 421 form.<br><br><br>",
+  "grossHint207": "Gross value of estate for probate in the UK in &pound;<br><br>This is the 'Gross estate in the UK passing under Will' on the form",
+  "grossHint421": "Gross value of assets for probate",
+  "grossHintNoIHT": "Gross value<br><br>This is the total value of the deceased's assets in their sole name",
```
Comment (AbdullahShahHMCTS) [inline app/resources/cy/translation/iht/probateestatevalues.json:10]: escape

File: app/resources/en/translation/iht/estateform.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{
+  "title": "Which IHT forms did you complete to report the estate&rsquo;s value?",
+  "question": "Which IHT forms did you complete to report the estate&rsquo;s value?",
+  "optionIHT400421": "IHT 400 and 421",
+  "optionIHT400421Hint": "Forms you completed because you have to pay inheritance tax",
+  "optionIHT207": "IHT 207",
+  "optionIHT207Hint": "Forms you completed because you do not have to pay inheritance tax because it&rsquo;s an excepted estate and the person who&rsquo;s died was domiciled abroad",
+  "errors": {
+    "ihtFormEstateId": {
+      "required": "Select which IHT forms you completed to report the estate’s value"
```
Comment (AbdullahShahHMCTS) [inline app/resources/en/translation/iht/estateform.json:10]: escape apostrophe

File: app/steps/ui/summary/includes/iht.njk [hunk 4]
```diff
@@ -140,4 +140,83 @@
...
+    ],
+    url = content.IhtEstateValues.url,
+    common = common,
+    alreadyDeclared = fields.summary.alreadyDeclared.value
+)}}
+
+{{checkanswer(
+    items = [
+        {
+            question: content.ProbateEstateValues.grossHint421,
```
Comment (AbdullahShahHMCTS) [inline app/steps/ui/summary/includes/iht.njk:183]: delete duplicates.
Also use new labels for the check your answers

File: test/component/iht/testEstateForm.js [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+    beforeEach(() => {
+        testWrapper = new TestWrapper('IhtEstateForm');
+    });
+
+    afterEach(() => {
+        testWrapper.destroy();
+    });
+
+    describe('Verify Content, Errors and Redirection', () => {
+        testCommonContent.runTest('ProbateEstateValues', null, null, [], false, {type: caseTypes.INTESTACY});
```
Comment (AbdullahShahHMCTS) [inline test/component/iht/testEstateForm.js:21]: Change to IhtEstateForm

File: app/resources/en/translation/iht/probateestatevalues.json [hunk 6]
```diff
@@ -1,32 +1,29 @@
...
+  "questionNoIHT": "What are the values of the estate for probate?",
+  "hint207": "<br>You can find these values on the completed IHT 207 form.<br><br><br>",
+  "hint421": "<br>You can find these values on the completed IHT 421 form.<br><br><br>",
+  "grossHint207": "Gross value of estate for probate in the UK in &pound;<br><br>This is the 'Gross estate in the UK passing under Will' on the form",
+  "grossHint421": "Gross value of assets for probate",
+  "grossHintNoIHT": "Gross value in &pound;<br><br>This is the total value of the deceased's assets in their sole name",
+  "netHint207": "Net value of the estate for Probate in &pound;",
+  "netHint421": "Net value of the estate for Probate",
+  "netHintNoIHT": "Net value in &pound;<br><br>Take away any debts in the deceased's sole name from the gross value",
+  "poundSymbol": "&pound;",
```
Comment (AbdullahShahHMCTS) [inline app/resources/en/translation/iht/probateestatevalues.json:14]: add label for check answers

**Review by AbdullahShahHMCTS — DISMISSED** [review]
Approved

**Review by itoulou — APPROVED** [review]

**Pull request merged**